### PR TITLE
Fix doctest tracebacks

### PR DESCRIPTION
**Pull request by CaedenPH**
### Describe your change:
Replace `File "/usr/bin..."` with `...` 

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: ciphers/decrypt_caesar_with_chi_squared.py [hunk 1]
```diff
@@ -122,6 +122,7 @@ def decrypt_caesar_with_chi_squared(
 
     >>> decrypt_caesar_with_chi_squared(12)
     Traceback (most recent call last):
+    ...
```
Comment (cclauss) [inline ciphers/decrypt_caesar_with_chi_squared.py:125]: Please indent four spaces like the doctest docs do.

File: quantum/superdense_coding.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def superdense_coding(bit_1: int = 1, bit_2: int = 1) -> qiskit.result.counts.Co
     # measure the circuit
     quantum_circuit.measure(qr, cr)
 
-    backend = Aer.get_backend("qasm_simulator")
+    backend = Aer.get_backend("aer_simulator")
```
Comment (cclauss) [inline quantum/superdense_coding.py:95]: This change was already merged?

Comment (CaedenPH) [reply quantum/superdense_coding.py:95]: > This change was already merged?

Must've forgotten to fetch

**Comment by CaedenPH** [comment]
@cclauss Should I go through every doctest and indent each `...` by a tab? Lots of the doctests in this repo follow 
```py
>>> x
Traceback (most recent call last):
...
```

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Optional to modify all files.  I think the adoption of `pyproject.toml` is a higher priority.

**Pull request merged**